src/executables/csql.c

```diff
@@ -3330,3 +3358,161 @@ get_host_ip (unsigned char *ip_addr)
 
   return 0;
 }
+
+static int
+csql_connect (char *argument, CSQL_ARGUMENT * csql_arg)
+{
+  /*dbname can be stored DB name+ @ + remote hostname + \0 */
+  char dbname[MAXLOGNAME + CUB_MAXHOSTNAMELEN + 2];
+  char username[DB_MAX_USER_LENGTH + 1];
+  const char *delim = " @\n";
+  char *save_ptr_strtok = NULL;
+  char *db_name_ptr = NULL;
+  char *user_name_ptr = NULL;
+  char *host_name_ptr = NULL;
+  char *p = NULL;
+  int error_code;
+  CSQL_ARGUMENT csql_new_arg;
+
+  /*verify @hostname */
```

Comment:
Check whether argument is not NULL first, if omitted it could be coredump when user typed just
;connect without arguments (I think I give you the comment for this)
csql> ;connect

```
if (argument == NULL)
  {
    ...
  }
```

Reply:
Right. It occured core dump. I will find any solution about this.
Thank you.
(error msg, function flow, etc)